src/test/resources/nodes_dn.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Is this change necessary?

Reply:
There are tests in src/test/java/com/amazon/opendistroforelasticsearch/security/SecurityAdminTests.java referencing securityconfig in main package instead of tests resources.

node_dn file was missing in the test resources hence added it.